packages/mobile/src/set-clock/SetClock.tsx

```diff
@@ -5,18 +5,24 @@ import colors from '@celo/react-components/styles/colors'
 import { fontStyles } from '@celo/react-components/styles/fonts'
 import * as React from 'react'
 import { withNamespaces, WithNamespaces } from 'react-i18next'
-import { Image, StyleSheet, Text, View } from 'react-native'
+import { Image, Platform, StyleSheet, Text, View } from 'react-native'
 import * as AndroidOpenSettings from 'react-native-android-open-settings'
 import { componentWithAnalytics } from 'src/analytics/wrapper'
 import { Namespaces } from 'src/i18n'
 import clockIcon from 'src/images/clock-icon.png'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
 import { getLocalTimezone, getRemoteTime } from 'src/utils/time'
 
 export class SetClock extends React.Component<WithNamespaces> {
   static navigationOptions = { header: null }
 
   goToSettings = () => {
-    return AndroidOpenSettings.dateSettings()
+    if (Platform.OS === 'android') {
+      return AndroidOpenSettings.dateSettings()
+    } else {
+      navigate(Screens.WalletHome)
```

Comment:
Can you leave a TODO here.
@jeanregisser do we already have a task tracking implementation for this on iOS?

Reply:
Good catch, I've just pushed the iOS implementation :+1: